### DISCUSSION TITLE
How can I get all the achievements in the gitHub?

### DISCUSSION THREAD

**Discussion by amx122** [community/community#50212] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How can I get all the achievements in the gitHub?

**Comment by neutrinomuon** [comment]
Hello, I think one of the best links is:

https://github.com/Schweinepriester/github-profile-achievements

Where you can find a detailed explanation of how to get achievements. However, there are some achievements that are no longer possible.

I hope this helps :)!
